README.md

```diff
@@ -43,9 +43,15 @@ Users are welcome to inquire about the status of particular snippets on the [dis
 
 To integrate this with your project, use Nix. We work against the main branch of Plutarch, so you will have to use it also. See the `flake.nix` file for more details.
 
-Liqwid uses [`liqwid-nix`](https://github.com/Liqwid-Labs/liqwid-nix) to set up its projects. 
-You are not required to use `liqwid-nix` to use this library, but you are welcome to check
-check it out.
+Liqwid uses [`liqwid-nix`](https://github.com/Liqwid-Labs/liqwid-nix) to set up
+its projects.  You are not required to use `liqwid-nix` to use this library, but
+you are welcome to check check it out.
+
+# Standards
+
+LPE code base follows a strict standards to increase consistency, to minimize
+legacy impacts, to use proper automated means, and more. The standard document 
```

Comment:
'the impact of legacy, ...'
'The standards document...'

I'm not sure what you mean by 'use proper automated means'.